parameterized_api_calls.yml

```diff
@@ -0,0 +1,23 @@
+version: '2.0'
+workflow:
+  tasks:
+    task_1:
+      action: civis.import
+      input:
+        sync_type: Dbsync
+        is_outbound: False
+        source:
+          credential_id: cred1
+          remote_host_id: db1
+        destination:
+          credential_id: cred2
+          remote_host_id: db2
+        syncs:
+        - source:
```

Comment:
this section needs to be indented

Reply:
either way is valid, I believe.

Reply:
Huh, gave it whirl - it's valid yaml.

Stylistically I prefer the indentation as it makes it clear the list is a child of syncs.